person.rb

```diff
@@ -13,6 +13,7 @@ def initialize(age, name, parent_permission)
     @age = age
     @parent_permission = parent_permission
     @nameable = nameable
+    @rental = []
   end
 
```

Comment:
- [ ] Kindly add the `add_rental` method to achieve the following relation `person has many rentals`. It will be something like this:
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```